netlify adapter docs: add warning on redirects file placement (#1596)
sveltejs/kit: packages/adapter-netlify/README.md

### Using Netlify Redirect Rules

> [Netlify docs here](https://docs.netlify.com/routing/redirects/)

> Netlify supports declarative redirects via a `_redirects` file placed in your `publish` directory (usually "`build`", as configured in your `netlify.toml` above. NOT in your project root next to `netlify.toml`).

Reviewer: the `build` directory is generated by SvelteKit, so I wouldn't think you could put it there. I thought on the last issue it was said to go in `static`?

Reviewer 2: Yes! It is `static/` unless you configured `kit.files.assets` to something else in the [project configuration](https://kit.svelte.dev/docs#configuration-files).

Author: Yeah my bad... 

```suggestion
Netlify supports declarative redirects via a `_redirects` file. Place it in the `static/` folder, NOT in your project root next to `netlify.toml`.
```

During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:

> - `[[redirects]]` in `netlify.toml` will never match as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order). So always put your rules in the [`_redirects` file](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file).

Author: just fyi - i think this is inaccurate - the doc actually says "Rules in the _redirects file are always processed first, followed by rules in the Netlify configuration file."

in other words you CAN put stuff in netlify.toml, AND in _redirects, its just that the latter overrides the former in cases of conflict.

Reviewer: I believe it's true when you consider the catch-all route that gets added
